lib/person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly add  `  @id = Random.rand(1..1000)`   to your instance variables defined 👍🏿

Reply:
Hi @prolajumokeoni! Thank you for your feedback. I'm doing just that inside my initialize method, isn't that okay?